radio/src/gyro.cpp

```diff
@@ -79,7 +79,7 @@ void Gyro::wakeup()
 
   int16_t gx = values[0];
   int16_t gy = values[1];
-  int16_t gz = values[2];
+  // int16_t gz = values[2];
```

Comment:
lol... already had this in another branch but hadn't come back to clean up more... thanks for getting rid of this one :)